### PR TITLE
Modifying DRISE runner to save all images per object

### PR DESCRIPTION
**Pull request by natalie-isak**
## Description

Removing logic to plot all saliency maps for a given image. Instead, creates separate plots per object.

### CODE REVIEW

File: python/vision_explanation_methods/DRISE_runner.py [hunk 1]
```diff
@@ -208,39 +108,18 @@ def get_drise_saliency_map(
...
-            print("loading user fastercnn model")
-            model = get_instance_segmentation_model(numclasses)
-            model.load_state_dict(
-                torch.load(modellocation, map_location=device))
-    else:
-        if modellocation:
-            print("loading any user model")
-            model.load_state_dict(
-                torch.load(modellocation, map_location=device))
+        print("using pretrained fastercnn model")
```
Comment (imatiach-msft) [inline python/vision_explanation_methods/DRISE_runner.py:111]: remove print

**Comment by Advitya17** [comment]
Several changes in this PR are quite similar to !18.

Can follow my comments from there and make changes here. Would recommend to try keeping code changes of different PRs independent to avoid repetition of work, merging older version of code, better separation of concerns etc.

File: python/vision_explanation_methods/DRISE_runner.py [hunk 2]
```diff
@@ -182,10 +170,4 @@ def get_drise_saliency_map(
...
         )
 
-        if num_detections > 1 or type(axis) == list:
-            axis[i] = plot_img_bbox(axis[i], box, str(label), 'r')
-        else:
-            axis = plot_img_bbox(axis, box, str(label), 'r')
-
-    fig.savefig(savename)
-    return fig, savename, label_list
+        fig.savefig(savename+str(i)+".jpg")
```
Comment (imatiach-msft) [inline python/vision_explanation_methods/DRISE_runner.py:173]: nit: move strings like jpg to constants

File: tests/test_vision_explanation.py [hunk 3]
```diff
@@ -16,13 +16,18 @@
 module_logger.setLevel(logging.INFO)
 
 try:
-    import torch
     import torchvision
     from torchvision.models.detection.faster_rcnn import FastRCNNPredictor
 except ImportError:
-    module_logger.debug('Could not import torch packages, required' +
+    module_logger.debug('Could not import torchvision packages, required' +
```
Comment (imatiach-msft) [inline tests/test_vision_explanation.py:22]: nit: required is using a pytorch computer **vision model**?

**Review by imatiach-msft — APPROVED** [review]

**Pull request merged**